### PR TITLE
 Username fixed on google sign in

### LINKED ISSUE #22: User name not same
**Describe the bug**
When a user first sign-in using email & password and later by Googe, the username in the Navigation drawer and Profile section is different.

**To Reproduce**
1. Sign in for the first time using email & password.
2. Log out and sign in again but this time using Google sign in.
3. Check username in the Navigation drawer and Profile section.

**Expected behavior**
The username should be the same everywhere.

**GIF**

googel_email

**Smartphone (please complete the following information):**
 - Device: Redmi 4
 - OS: Android Nougat

### PR DESCRIPTION
**Pull request by prajwal72**
LoginActivity.java: Profile and Navigation header were showing different user name.

Fixes #22

### CODE REVIEW

**Comment by sonusourav** [comment]
First of all, rebase your master branch. It can be seen that the last pull request is included in this also.

File: app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java [hunk 1]
```diff
@@ -195,10 +195,9 @@ public boolean validateUserInput(String signUpEmail,String signUpUsername,String
             return false;
         }
 
-
-
 //        String[] split = signUpEmail.split("@");
 //        String domain = split[1]; //This Will Give You The Domain After '@'
+
```
Comment (sonusourav) [inline app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java]: Remove this extra line.

File: app/src/main/java/com/iitdh/sonusourav/instigo/Account/LoginActivity.java [hunk 2]
```diff
@@ -380,13 +384,28 @@ public void onComplete(@NonNull Task<Void> task) {
...
+                                            user.updateProfile(request);
+                                        }
+
+                                        @Override
+                                        public void onCancelled(@NonNull DatabaseError databaseError) {
+
+                                        }
+                                    });
+
                                     Toast.makeText(LoginActivity.this, " Signed In as " + user.getDisplayName(), Toast.LENGTH_SHORT).show();
```
Comment (sonusourav) [inline app/src/main/java/com/iitdh/sonusourav/instigo/Account/LoginActivity.java]: Still when the user login in for the first time using Google sign-in after he has logged in earlier with Email & password, Toast uses Google username.

Also, in the nav bar, Google username is used for the first time when signed in using Google. After refreshing, profile username is used.

**Review by sonusourav — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/iitdh/sonusourav/instigo/Account/LoginActivity.java (hunk 2 above)
Comment (prajwal72) [reply app/src/main/java/com/iitdh/sonusourav/instigo/Account/LoginActivity.java]: Done

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <Objective-C-extensions>
+      <file>
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Import" />
```
Comment (sonusourav) [inline .idea/codeStyles/Project.xml]: Add this file to gitignore.

File: app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java [hunk 4]
```diff
@@ -128,6 +128,7 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                                     Toast.makeText(RegisterActivity.this, "The user account has been disabled by an administrator.", Toast.LENGTH_LONG).show();
                                     break;
 
+
```
Comment (sonusourav) [inline app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java]: Remove this line.

File: app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java [hunk 5]
```diff
@@ -137,11 +138,17 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                                     registerPass.setError("The password is invalid it must 6 characters at least");
                                     registerPass.requestFocus();
                                     break;
+
```
Comment (sonusourav) [inline app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java]: Remove this line.

File: app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java [hunk 6]
```diff
@@ -137,11 +138,17 @@ public void onComplete(@NonNull Task<AuthResult> task) {
...
                             }
                         }
                     }
+
+
```
Comment (sonusourav) [inline app/src/main/java/com/iitdh/sonusourav/instigo/Account/RegisterActivity.java]: Remove extra lines everywhere. A small piece of advice, while committing, just see the difference in the changed files and remove which should not be there.

**Review by sonusourav — CHANGES_REQUESTED** [review]
Also in line no 279 in Login Activity, just add the user name who logs in. Right now, its just "Signed in successfully". Change it to " Signed in as `username`"

**Review by sonusourav — APPROVED** [review]

**Pull request merged**